content/commands/hexpire/index.md

```diff
@@ -129,10 +127,10 @@ redis> HGETALL mykey
 
 One of the following:",
 * [Array reply](../../develop/reference/protocol-spec#arrays). For each field:
-    - [Integer reply](../../develop/reference/protocol-spec#integers): `-2` if no such field(s) exist in the provided hash key.
+    - [Integer reply](../../develop/reference/protocol-spec#integers): `-2` if no such field exists in the provided hash key.
     - [Integer reply](../../develop/reference/protocol-spec#integers): `0` if the specified NX | XX | GT | LT condition has not been met.
     - [Integer reply](../../develop/reference/protocol-spec#integers): `1` if the expiration time was set/updated.
-    - [Integer reply](../../develop/reference/protocol-spec#integers): `2` if the field(s) were deleted because of previously set expiration(s). This reply may also be returned when `HEXPIRE`/`HPEXPIRE` is called with 0 seconds/milliseconds or when `HEXPIREAT`/`HPEXPIREAT` is called with a past unix-time in seconds/milliseconds.
+    - [Integer reply](../../develop/reference/protocol-spec#integers): `2` if the field was deleted because of previously set expiration. This reply may also be returned when `HEXPIRE`/`HPEXPIRE` is called with 0 seconds/milliseconds or when `HEXPIREAT`/`HPEXPIREAT` is called with a past unix-time in seconds/milliseconds.
```

Comment:
" `2` if the field was deleted because of previously set expiration.  " is wrong. If the field already expired before the command was called - the reply would be -2.
When I wrote "2: field deleted because the specified expiration time is due" I meant just the two other possibilities (when `HEXPIRE`/`HPEXPIRE` is called with 0 seconds/milliseconds or when `HEXPIREAT`/`HPEXPIREAT` is called with a past unix-time in seconds/milliseconds.)

Reply:
unix --> Unix

Reply:
Changes in next push.